ui-tests/src/test/java/io/syndesis/qe/steps/integrations/IntegrationSteps.java

```diff
@@ -320,14 +320,4 @@ public void verifyTypeOfConnection(String type) {
 		SelenideElement connection = editPage.flowViewComponent().flowConnection(type).getElement().shouldBe(visible);
 		log.info("{} connection must be active", type);
 	}
-
-	/**
-	 * check you are on the correct editPage
-	 *
-	 * @param text
-	 */
-	@Then("^she is presented with \"([^\"]*)\" (page|editor)$")
```

Comment:
agree. 
just note: it would be good if these 'presented' steps are standardized. most suitable via using .validate() method.

Reply:
@Stefan365 As i wrote in your pull request Martin already did this.